obs-vscode-extension/server/src/server.ts

```diff
@@ -0,0 +1,80 @@
+import { createConnection, ProposedFeatures, TextDocuments, InitializeParams, TextDocumentSyncKind, TextDocumentChangeEvent, Position, Diagnostic, Range, DiagnosticSeverity } from "vscode-languageserver";
```

Comment:
Awfully long. Maybe just import everything from this package?

Reply:
Unfortunately there is no such mechanism to do so in typescript/javascript. The alternative is to import the entire namespace which can become very verbose